specification/quantum/data-plane/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -0,0 +1,552 @@
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/jobDetailsList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get list of jobs in a Quantum Workspace": {
+            "$ref": "./examples/jobList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Quantum/workspaces/{workspaceName}/jobs/{jobId}": {
+      "get": {
+        "operationId": "Jobs_Get",
+        "description": "Get job by id",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "Get a job's metadata": {
+            "$ref": "./examples/jobGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/restError"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "Jobs_Put",
+        "description": "Create a job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/jobIdParameter"
+          },
+          {
+            "name": "jobDefinition",
+            "description": "The complete metadata of the job to submit.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          }
+        ],
+        "x-ms-examples": {
+          "Submit a new job for execution to a Quantum Workspace": {
+            "$ref": "./examples/jobPut.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/jobDetails"
+            }
+          },
+          "201": {
```

Comment:
Does this operation support LRO protocol?

Reply:
Not yet, also in our TODO list.